x-pack/plugins/enterprise_search/public/applications/enterprise_search/index.scss

```diff
@@ -24,6 +24,7 @@
     @include euiBreakpoint('m', 'l', 'xl') {
       margin-bottom: $euiSizeL;
     }
+    background-color: $euiPageBackgroundColor; // Fails axe contrast checks without this (due to the background image on the page)
```

Comment:
cc @daveyholler for approval on this SCSS change, it's not strictly necessary to get tests to pass but it removes an axe-devtools complaint on the page so I figured why not add it 🤔

Reply:
Caroline's removing the need for this change in #107709, so I'll remove this change